packages/ui/public/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <meta name="theme-color" content="#000000" />
+
+    <link rel="manifest" href="/manifest.json" />
+
+    <script src="https://js.stripe.com/v3"></script>
+
+    <link href="https://fonts.googleapis.com/css?family=Literata|Nunito&display=swap" rel="stylesheet">
+
+    <title>Scorekeep</title>
+
+    <style>
+      html, body, #root {
+        position: relative;
+        height: 100%;
+        width: 100%;
+        margin: 0;
+        padding: 0;
+
+        font-family: 'Nunito', sans-serif;
+      }
+
+      button {
+        border-radius: 1px;
+        border: none;
+        font-family: 'Nunito', sans-serif;
+        cursor: pointer;
+      }
+
+      h1, h2, h3, h4, h5 {
+        font-family: 'Literata', sans-serif;
+      }
+
+      a, a:visited {
+        color: deepskyblue;
+      }
+
+      #root {
+        background: #111;
+        color: #eee;
+      }
+
+      label {
+        color: #eee;
+      }
+
+      #root * {
+        box-sizing: border-box;
+      }
+    </style>
+
+    <style>
+      #portal-root, #portal-root > div {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100vw;
+        height: 100vh;
+        pointer-events: none;
```

Comment:
dunno why this is here / if its necessary

Reply:
Removing

Reply:
Is basically for a createPortal from react, since we still do not use modals I am removing both node and styles